### PR TITLE
VSCode settings added

### PR DESCRIPTION
**Pull request by Mark-Giesen**
Added VSCode .gitignore settings and launch.json.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Huidige bestand",
```
Comment (StephanEggermont) [inline .vscode/launch.json:8]: are you sure you want that to be Dutch?

**Comment by Felienne** [comment]
Hi @Mark-Giesen!

It is great to add the vscode folder to the gitignore, thanks! But we don't want to add the folder itself too, because these two changes will take effect together, so you add the folder before adding the instruction to ignore it. Can you remove your vscode files and the folder from the PR?

File: .vscode/launch.json (hunk 1 above)
Comment (Mark-Giesen) [reply .vscode/launch.json:8]: Oops, no! Good catch. The file was generated, didn't check the language.

**Comment by Mark-Giesen** [comment]
The vscode folder is not added to .gitignore, I belive, just the files in it, except the re-included ones. Or am I misreading your comment?

**Comment by Felienne** [comment]
> The vscode folder is not added to .gitignore, I believe, just the files in it, except the re-included ones. Or am I misreading your comment?

Yeah fair enough, what I meant to say is that we shouldn't include the files in the folder because, these are your files.

**Comment by Felienne** [comment]
Removed it for you :)

**Review by Felienne — APPROVED** [review]

**Pull request merged**